### PR TITLE
Add a Creative Commons Attribution Share Alike 4.0 International License

### PR DESCRIPTION
**Pull request by MarkBennett**
Screenshot from 2020-04-02 21-58-03

This adds it to the `LICENSE.txt` as well as the footer on all pages.

Details on the license at:

https://creativecommons.org/licenses/by-sa/4.0/

### CODE REVIEW

**Comment by MarkBennett** [comment]
@ajyong @benpryke either of you around for a review?

File: _layouts/base.html [hunk 1]
```diff
@@ -62,6 +62,7 @@
           </li>
           <li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
           <li><a href="{{ '/policies/privacy-policy.html' | relative_url }}">Privacy Policy</a></li>
+          <li><a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">CC Attrib-SA 4.0 License</a>.</li>
```
Comment (benpryke) [inline _layouts/base.html]: Looks like a copy/paste issue, but `<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">` is duplicated here. For future reference, would you prefer if I added a quick change via the GitHub web UI here as another commit or would you prefer to eg fix and squash yourself?

**Review by benpryke — CHANGES_REQUESTED** [review]

**Comment by MarkBennett** [comment]
@benpryke Good catch thanks! I'll fix that right now.

I'm good with web UI or the local dev. VS Code is usually what I use, but I usually just pick what's at hand.

File: _layouts/base.html (hunk 1 above)
Comment (MarkBennett) [reply _layouts/base.html]: Fixed!

**Comment by MarkBennett** [comment]
Ready for another review @benpryke. I fogot to commit the change to delete the extra `<a>`. It's there now.

**Review by benpryke — APPROVED** [review]
lgtm!

**Pull request merged**